Resources/config/schema/swiftmailer-1.0.xsd

```diff
@@ -66,6 +66,10 @@
     </xsd:restriction>
   </xsd:simpleType>
 
+  <xsd:simpleType name="delivery-address">
```

Comment:
this is not true. Symfony uses the local schema. the issue is that you defined a custom type, but you never use it